### PR TITLE
Create scorecard.yml

### PR DESCRIPTION
**Pull request by baristaGeek**
## Description
Adds the OSSF scorecard as our other 2 repos

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)  
- [ ] New feature (non-breaking change which adds functionality)  
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)  
- [x] Documentation update  
- [x] Chore: cleanup/renaming, etc  
- [ ] RFC  

## Acceptance
- [x] I have read [the Contributing guidelines](CONTRIBUTING.md)
- [x] I have read the [Code of Conduct](CODE_OF_CONDUCT.md)

### CODE REVIEW

File: .github/workflows/scorecard.yml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+name: Scorecard supply-chain security
+on:
+  # For Branch-Protection check. Only the default branch is supported. See
+  # https://github.com/ossf/scorecard/blob/main/docs/checks.md#branch-protection
+  branch_protection_rule:
+  # To guarantee Maintained check is occasionally updated. See
+  # https://github.com/ossf/scorecard/blob/main/docs/checks.md#maintained
+  schedule:
+    - cron: '27 9 * * 4'
```
Comment (EstebanDalelR) [inline .github/workflows/scorecard.yml:13]: Why on Thursday at 9:27 AM?

File: .github/workflows/scorecard.yml [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+          publish_results: true
+
+      # Upload the results as artifacts (optional). Commenting out will disable uploads of run results in SARIF
+      # format to the repository Actions tab.
+      - name: "Upload artifact"
+        uses: actions/upload-artifact@3cea5372237819ed00197afe530f5a7ea3e805c8 # v3.1.0
+        with:
+          name: SARIF file
+          path: results.sarif
+          retention-days: 5
```
Comment (EstebanDalelR) [inline .github/workflows/scorecard.yml:66]: Runs weekly but only does retention for five days?

File: .github/workflows/scorecard.yml (hunk 1 above)
Comment (baristaGeek) [reply .github/workflows/scorecard.yml:13]: This yaml is a default from the GH marketplace

File: .github/workflows/scorecard.yml (hunk 2 above)
Comment (baristaGeek) [reply .github/workflows/scorecard.yml:66]: This yaml is a default from the GH marketplace

**Review by EstebanDalelR — APPROVED** [review]

**Pull request merged**